<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Задайте начальную точку:</string>
    </property>
    <widget class="QDoubleSpinBox" name="zero">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculateBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберете функцию:</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>242</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="function1">
        <property name="text">
         <string>f(x)=(x-1)^2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="function2">
        <property name="text">
         <string>f(x)=4x^3-8x^2-11x+5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="function3">
        <property name="text">
         <string>f(x)=x+(3/x^2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="function4">
        <property name="text">
         <string>f(x)=(x-2.5)/(4-x^2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="function5">
        <property name="text">
         <string>f(x)=-sin(x)-(sin(3x))/3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="function6">
        <property name="text">
         <string>f(x)=-2sin(x)-sin(2x)-(2sin(3x))/3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберете метод:</string>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="method1">
        <property name="text">
         <string>Метод Фибоначи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="method2">
        <property name="text">
         <string>Метод Пауэлла</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Xmin</string>
    </property>
    <widget class="QLineEdit" name="xmin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Интервал</string>
    </property>
    <widget class="QLineEdit" name="interval">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
